modules/nf-core/software/bwamem2/index/main.nf

```diff
@@ -1,13 +1,15 @@
+// Import generic module functions
 include { initOptions; saveFiles; getSoftwareName } from './functions'
 
 params.options = [:]
 def options    = initOptions(params.options)
 
 process BWAMEM2_INDEX {
-    tag "${fasta}"
-
-    publishDir params.outdir, mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:"false") }
+    tag "$fasta"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
 
     conda (params.enable_conda ? "bioconda::bwa-mem2=2.1" : null)
```

Comment:
There was a new release for bwamem2. Have you checked wether the new conda build and containers are already available?

Reply:
Not at all, I'll check this out